File: lab19/task5.java
// ...existing code...
public class task5 {

    // Simple Car class with brand, model and year attributes
    static class Car {
        private String brand;
        private String model;
        private int year;

        public Car(String brand, String model, int year) {
            this.brand = brand;
            this.model = model;
            this.year = year;
        }

        // Prints the car details
        public void displayDetails() {
            System.out.println("Brand: " + brand + ", Model: " + model + ", Year: " + year);
        }
    }

    // Demo main
    public static void main(String[] args) {
        Car car = new Car("Toyota", "Corolla", 2020);
        car.displayDetails();
    }
}
// ...existing code...

File: lab19/task5.py
# ...existing code...
class Car:
    """Simple Car class with brand, model and year attributes."""
    def __init__(self, brand: str, model: str, year: int):
        self.brand = brand
        self.model = model
        self.year = year

    def display_details(self) -> None:
        """Prints the car details."""
        print(f"Brand: {self.brand}, Model: {self.model}, Year: {self.year}")

if __name__ == "__main__":
    # Demo
    car = Car("Toyota", "Corolla", 2020)
    car.display_details()
# ...existing code...